apps/steercode-app-electron/src/renderer/src/lib/models/types/user.type.ts

```diff
@@ -0,0 +1,22 @@
+import type { RepositoryOption } from './conversation.type';
+
+export type StripeInfo = {
+  customer_id: string;
+  subscription_id: string | null;
+  subscribed_until: string | null;
+};
+
+export type User = {
+  id: string;
+  projects: RepositoryOption[];
+  stripe: StripeInfo | null;
+  isPremium: () => boolean;
```

Comment:
this function does not exist on the User object. either remove it from the type definition or implement it please